src/SearchPlugin.ts

```diff
@@ -5,14 +5,14 @@ import { PluginOptions } from "./types";
 
 export function createSearchPlugin(app: App, options: PluginOptions) {
   const { router, getApi } = options;
-  console.log("search plugin created");
+
   // Register plugin components
   app.component("CrucibleSearch", CrucibleSearch);
   app.component("DisplayResult", DisplayResult);
   app.provide("$router", router);
   app.provide("$getApi", getApi);
   // Add plugin routes to the existing router
-  router.addRoute({ path: "/search-in-tag/:tag", component: DisplayResult });
+  router.addRoute({ path: "/search-in-tag", component: DisplayResult });
```

Comment:
Is this going to be the url, why not just `search`?

Reply:
changed to search
was thinking about: search for tag/label/content